### DISCUSSION TITLE
Help me！！！

### DISCUSSION THREAD

**Discussion by kana-takahashiii** [community/community#33094] [New to GitHub] [answered]
**Please help！！！**

I registered with github and got this message.

「Your account has been flagged.
If you believe this is a mistake, contact support to have your account status reviewed.」

I contacted support to cancel the account, but received no response.
Will the repository disappear if I don't? I am worried. What should I do?
Please help me, I am a beginner.
Thank you in advance.

**Comment by queenofcorgis** [comment, marked as answer]
Hi there - please be patient as Support is the only team that can assist in this matter, the Community is unable to help here. Best of luck!
